src/tools/run-make-support/src/fs_helpers.rs

```diff
@@ -0,0 +1,71 @@
+use std::io;
```

Comment:
I don't think we need two modules, it's a bit confusing to decide between `fs_helpers` and `fs_wrapper`. I'd create just a single module, and we can also just name it `fs`, to make tests shorter, seeing `fs_wrapper` everywhere is a bit distracting.